20.Megatexture/main.cpp

```diff
@@ -709,7 +713,7 @@ APP_CONSTRUCTOR(MegaTextureApp)
         auto commandBuffer = commandBuffers[0];
 
         commandBuffer->reset(nbl::video::IGPUCommandBuffer::ERF_RELEASE_RESOURCES_BIT);
-        commandBuffer->begin(IGPUCommandBuffer::EU_NONE);
+        commandBuffer->begin(video::IGPUCommandBuffer::EU_NONE);
```

Comment:
There is an error which causes this example to not compile
```
'nbl::asset::ICPUShader::ICPUShader(const nbl::asset::ICPUShader &)': cannot convert argument 1 from '_Ty' to 'const nbl::asset::ICPUShader &'
```

Reply:
huh? this is an `IGPUCommandBuffer::begin` ?